doc/source/how-to-run-flower-on-azure.rst

```diff
@@ -0,0 +1,197 @@
+.. |authenticate_supernodes| replace:: Authenticate Supernodes
+
+.. _authenticate_supernodes: how-to-authenticate-supernodes.html
+
+.. |enable_tls_connections| replace:: Enable TLS Connections
+
+.. _enable_tls_connections: how-to-enable-tls-connections.html
+
+.. |flower_architecture_link| replace:: Flower Architecture
+
+.. _flower_architecture_link: explanation-flower-architecture.html
+
+Run Flower on Azure
+===================
+
+.. note::
+
+    There are many ways to deploy Flower on Microst Azure. The instructions provided in
+    this guide is just a basic walkthrough, step-by-step guide on how to quickly setup
+    and run a Flower application on a Federated Learning environment on Microst Azure.
+
+In this how-to guide, we want to create a Federated Learning environment on Microst
+Azure using three Virtual Machines (VMs). From the three machines, one machine will be
+used as the Federation server and two as the Federation clients. Our goal is to create a
+Flower federation on Microst Azure where we can run Flower apps from our local machine,
+e.g., laptop.
+
+On the Federation server VM we will deploy the long-running Flower server
+(``SuperLink``) and on the two Federation client VMs we will deploy the long-running
+Flower client (``SuperNode``). For more details For more details regarding the
+``SuperLink`` and ``SuperNode`` concepts, please see the |flower_architecture_link|_ .
+
+Azure VMs
+---------
+
+First we need to create the three VMs configure their Python environments, and inbound
+networking rules to allow cross-VM communication.
+
+VM Create
+~~~~~~~~~
+
+Assuming we are already inside the Microst Azure portal, we navigate to the ``Create``
+page and we select ``Azure virtual machine``. In the new page that opens up, for each VM
+we fill in the following fields:
+
+- **Virtual machine name**: for instance, for the server machine we can use
+  ``flower-server`` and for the clients, ``flower-client-1``, ``flower-client-2``,
+  respectively.
+- **Image**: for this guide, we will use ``Ubuntu Server 24.04 - x64 Gen2 LTS``
+- **Size**: for this guide, we will use ``Standard_D2s_v3 - 2 vcpus, 8GiB memory``
+
+.. tip::
+
+    For resource group, we can create a new group and assign it to all VMs.
```

Comment:
Wouldn't all this benefit from a screenshot? or do you think it's better like this?

Reply:
I though the same. However, if we add a screenshot then we might have to capture the entire portal and mark the specific sections, which might be illegible. We can add a screenshot though if need to.

Reply:
In the last commit I placed this information inside a table to improve readability. Maybe that looks a bit better.